src/java/org/apache/cassandra/schema/Schema.java

```diff
@@ -533,13 +535,13 @@ private synchronized void updateVersion(UUID version)
     }
 
     /**
-     * Clear all KS/CF metadata and reset version.
+     * Clear all KS/CF metadata and reset version. Not to be ever used in production code
      */
-    public synchronized void clear()
+    @VisibleForTesting
+    public synchronized void clearUnsafe()
```

Comment:
My IDE didn't find any usages of this method. Also, searching on the commit I didn't find any?